Core/Core/Common/CommonModels/API/APIGraphQLRequestable.swift

```diff
@@ -34,25 +34,25 @@ public protocol APIPagedRequestable: APIRequestable where Response: PagedRespons
     func nextPageRequest(from response: Response) -> NextRequest?
 }
 
-protocol APIGraphQLRequestable: APIRequestable {
+public protocol APIGraphQLRequestable: APIRequestable {
     associatedtype Variables: Codable, Equatable
 
     static var query: String { get }
     static var operationName: String { get }
     var variables: Variables { get }
 }
 
-extension APIGraphQLRequestable {
-    public var method: APIMethod {
+public extension APIGraphQLRequestable {
+    var method: APIMethod {
         .post
     }
-    public var path: String {
+    var path: String {
         "/api/graphql"
     }
-    public static var operationName: String {
+    static var operationName: String {
         "\(self)"
     }
-    public var body: GraphQLBody<Variables>? {
+    var body: GraphQLBody<Variables>? {
         GraphQLBody(query: Self.query, operationName: Self.operationName, variables: variables)
```

Comment:
Keeping the properties public didn't work? Moving the extension to public is agains our [guidelines](https://github.com/instructure/canvas-ios/blob/master/.github/guidelines/swift-style-guide.md#extension-access-levels).